Assets/Scripts/Models/Tile.cs

```diff
@@ -41,6 +41,7 @@ public Tile(int x, int y, int z)
         Characters = new List<Character>();
         MovementModifier = 1;
         Utilities = new Dictionary<string, Utility>();
+        ReservedAsWorkSpotBy = new HashSet<Furniture>();
```

Comment:
This is a HashSet so that multiple furniture can use the same Work Spot? Right now is used as a boolean.